### DISCUSSION TITLE
getting in after having forgotten my PW

### DISCUSSION THREAD

**Discussion by jmw512** [mastodon/mastodon#22986] [Q&A] [answered]
Silly problem:
I joined already but forgot to note my PW. Now I'm trying to sign in but to change my PW, they want me to include my forgotten PW.
How do I circumvent this?
Thanks in advance
Justin

**Comment by nightpool** [comment, marked as answer]
You can use the "Forgot Password" link at the bottom of the sign in page to reset your password.
